docs/_posts/2017-11-30-nullannotations.md

```diff
@@ -6,13 +6,13 @@ image: "2017-11-30-nullannotations.png"
 published: true
 ---
 
-Nullpointer Exceptions (aka NPE's) are pretty nasty since they are `RuntimeException`s and thus a developers does not immediately notice them while coding in their IDE.
+Nullpointer Exceptions (aka NPE's) are pretty nasty since they are `RuntimeException`s and thus developers do not immediately notice them while coding in their IDE.
 Hence, data-flow analysis where written that show potential accesses to nullpointers before they occur at runtime. Our project decided to make use of these ananlyses and get rid of (some) nullpointers.
 
 <!--more-->
 
 Two popular projects that offer such an analysis are the [Checker Framework](https://checkerframework.org) and the [Eclipse JDT compiler](https://wiki.eclipse.org/JDT_Core/Null_Analysis).
-In order to be obtain better results from these anylyses, developers can make use of `annotations` in the code.
+In order to obtain better results from these anylyses, developers can make use of `annotations` in the code.
```

Comment:
`analysis`

Reply:
Its analyses (plural) see: https://www.dict.cc/?s=Analysen

But nevertheless I had a typo in there :P